dashboard/lib/services/csta_enrollment.rb

```diff
@@ -54,8 +54,8 @@ def self.submit(first_name:, last_name:, email:, school_district_name:,
         "submission[17_state]" => get_us_state_abbr(state, true),
         "submission[17_zip]" => zip,
         "submission[19]" => "Yes, I provide my consent.",
-        "submission[20]" => professional_role,
-        "submission[21]" => grades_teaching
+        "submission[25]" => professional_role,
```

Comment:
Just confirming that 20 -> 25 and 21 -> 22? The change of order is slightly unexpected so just double checking!

Reply:
Correct - the role is now 25 and the grade is 22.